mongoengine/queryset/base.py

```diff
@@ -19,13 +18,19 @@
 from mongoengine.common import _import_class
 from mongoengine.connection import get_db
 from mongoengine.context_managers import switch_db
-from mongoengine.errors import (InvalidQueryError, LookUpError,
-                                NotUniqueError, OperationError)
+from mongoengine.errors import (
```

Comment:
Why the change? I think personally I prefer the more compact original version of this line.